lib/ads/interstitial_ad_manager.js

```diff
@@ -1630,6 +1645,59 @@ shaka.ads.InterstitialAdManager = class {
   getInterstitials() {
     return Array.from(this.interstitials_);
   }
+
+
+  /**
+   * @param {!shaka.extern.AdInterstitial} interstitial
+   * @private
+   */
+  checkPreloadImages_(interstitial) {
+    if (this.preloadImages_.has(interstitial)) {
+      return;
```

Comment:
Maybe this should be configurable? Not sure does everybody wants to have DOM manipulation, esp. when `mode="preload"` does not work.

Reply:
https://caniuse.com/link-rel-preload --> It's not really a problem, if it doesn't support it, the browser ignores it

Reply:
I added ads.allowPreloadOnDomElements to config this